libs/store/src/lib/effects/race.ts

```diff
@@ -0,0 +1,168 @@
+import {
+  AppPlayerId,
+  AppPlayerProfiles,
+  AppRace,
+  AppRaceId,
+  AppTournament,
+  AppTournamentId,
+  AppTournamentState,
+} from '@razor/models';
+import {
+  calculateTimeoutTimer,
+  extractId,
+  extractIdType,
+  generateLeaderboard,
+  giveZeroPadding,
+} from '@razor/util';
+import { playerNotFound, tournamentNotFound } from '../loggers';
+import { raceNotFound } from '../loggers/race';
+import {
+  endCountdownPayload,
+  endRacePayload,
+  startCountdownPayload,
+} from '../payloads';
+import { Dispatch, RootState } from '../store';
+
+//TODO: should move this to server later
+const loadRacingText = async (): Promise<string> => {
+  const url = 'http://www.metaphorpsum.com/paragraphs/1/8';
+
+  return fetch(url)
+    .then(response => response.text())
+    .then(data => {
+      return data;
+    });
+};
+
+export const startCountdown = async (
+  dispatch: Dispatch,
+  payload: startCountdownPayload,
+  state: RootState,
+): Promise<void> => {
+  const {
+    tournamentId,
+    playerId,
+  }: { tournamentId: AppTournamentId; playerId: AppPlayerId } = payload;
+
+  if (!state.game.tournamentsModel[tournamentId])
+    tournamentNotFound(dispatch, tournamentId, `Started by: ${playerId}`);
```

Comment:
Why do we continue execution despite having failed an assertion?

Reply:
Fix in PR #14 under commit 2c90ad8bcbf7254f6da140cca1193fcea40da26e